currency-rates/internal/services/subscription.go

```diff
@@ -15,21 +15,21 @@ type SubscriptionDb interface {
 	GetEmails(ctx context.Context) ([]Email, error)
 }
 
-type SubscriptionSender interface {
-	Send(ctx context.Context, rate float64, emails []string) error
+type SubscriptionPublisher interface {
```

Comment:
As i wrote upper, this should be Abstract interface, not concrete .

Reply:
Done